src/components/toolbar/README.md

```diff
@@ -0,0 +1,59 @@
+# MdToolbar
+`MdToolbar` is a vertical aligned bar, which can hold the application title or actions.
+
+### Screenshots
+![Preview](https://cloud.githubusercontent.com/assets/4987015/13727769/6d952c78-e900-11e5-890a-ccfd46996812.png)
+
+## `<md-toolbar>`
+### Bound Properties
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `color` | `"primary" | "accent" | "warn"` | The color palette for the toolbar |
+
+### Notes
+By the default the `md-toolbar` will use the background palette.
+
+Multiple rows inside of a `md-toolbar` can be added by appending as many as needed `<md-toolbar-row>` elements.
+
+### Example
+```html
+<md-toolbar [color]="myColor">
```

Comment:
Start off with a simple example of the toolbar with just a single row (most common case), and then say something like "The toolbar also supports multiple rows" with the second example.